install/gcp/installer/util/utils.py

```diff
@@ -109,20 +109,46 @@ def _print_banner(border_symbol, edge_symbol, corner_symbol,
 
 
 def get_forseti_version():
-    """Get Forseti version from version file.
+    """Get the current Forseti version. If the current version is a branch
+    or a tag, we will return it. Otherwise we will read the __init__ file
+    for the latest version.
 
     Returns:
         str: The version.
     """
     version = None
+    return_code, out, err = run_command(
```

Comment:
Add a comment that this is checking for version by branch.  It's not obvious what the git symbolic-ref is doing.
```
# Check version by branch
```

Reply:
Updated.